cooperator/data/mail_template_data.xml

```diff
@@ -399,7 +399,7 @@
             ><![CDATA[
 <div style="font-family: 'Lucica Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; ">
 
-    <p>Hello ${object.subscription_request_ids[0].firstname},</p>
+    <p>Hello ${object.firstname},</p>
```

Comment:
The recipient may not have subscription requests.

Because this file is marked noupdate, clients need to fix this locally.